project/scripts/sbtBootstrappedTests

```diff
@@ -2,3 +2,34 @@
 
 # check that benchmarks can run
 ./project/scripts/sbt "dotty-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
+
+
+# setup for `dotc`/`dotr` script tests
+./project/scripts/sbt dist-bootstrapped/pack
+
+# check that `dotc` compiles and `dotr` runs it
+echo "testing sbt dotc and dotr"
+mkdir out/scriptedtest0
+./dist-bootstrapped/target/pack/bin/dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest0
```

Comment:
Any reason to use `dist-bootstrapped/**/dotc` instead of `bin/dotc`?

Reply:
I wanted to make sure I was testing that version of the script. We could also add some tests for `bin/dotc` in the future.

Reply:
`bin/dotc` just forwards to `dist-bootstrapped/target/pack/bin/dotc` and call `sbt dist-bootstrapped/pack` if needed :)

Reply:
Ok, I'll change it.